### DISCUSSION TITLE
Can I make the tooltip not show up on phone?

### DISCUSSION THREAD

**Discussion by ru88s** [saadeghi/daisyui#715] [Q&A] [answered]
It is convenient in PC view, but in phone it is a tap to view and it gets in the way.
Is there a solution?

**Comment by saadeghi** [comment, marked as answer]
[`tooltip` class](https://daisyui.com/components/tooltip/) is not responsive yet but I would consider making it responsive in the future updates.

For now you can use 2 elements and hide the one with `tooltip` on mobile size: https://play.tailwindcss.com/gDLjWCx0kE

**Reply by ru88s** [reply]
I would like to see responsive support.

I use tooltips a lot and it is hard to need two elements.
Thank you for the solution.

**Comment by fini** [comment]
@saadeghi I am also struggling with Tooltips on mobile, because of 'sticky hover' issue, which results in mobile browsers showing the tooltip whenever the input field is focused, unfortunately giving a very bad UX. I think I will attempt to make a fix and a PR to solve it. There are many ways this could be done, so not sure what it will end up as.

**Reply by saadeghi** [reply]
I'm trying to fix this in daisyUI version `3.0`  
I will let you know when it's fixed.
